src/components/Chip/Chip.scss

```diff
@@ -12,7 +12,7 @@ $vertical-padding: ($min-height - line-height(body) - rem(2px)) / 2;
     height: $height;
     min-width: $height;
     margin: 0;
-    padding: 0 $horizontal-padding;
+    padding: 0 ($horizontal-padding + .3) 0 0;
```

Comment:
@Farhan-emgage why is the right padding being increased and the left padding removed? I'm pretty sure this will break instances of the chip with image prop, and make the left portion of the newly added chipButtonContent unclickable.

Reply:
@weegrom  actually previously image / icon (left side) and content both was separate but now we place it in one button. so for that we had remove the left padding. and for right padding there are border between remove and content. so making space between them we had increase the right padding